### PR TITLE
Qfweiwei/update voice assistant test

### PR DESCRIPTION
**Pull request by qfweiwei**
## Purpose

Add hospitality skill test files that can be used with Voice Assistant Test tool

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Run tests with Voice Assistant Test tool

### CODE REVIEW

File: custom-commands/hospitality/test/hospitality_skill.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      "SingleConnection": true,
+      "Skip": false,
+	  "IgnoreActivities": [
+        {
+          "type": "event",
+        }
+      ]
+    }
+  ],
+  "SubscriptionKey": "fd0d3bd0b03f4e36a149e342b56d5004",
```
Comment (splumber) [inline custom-commands/hospitality/test/hospitality_skill.json:64]: Please replace these values with
Fake Speech Key: 89t7s45tyu2i7y9j908w345u57962eb2
Azure Region: westus2

Comment (dargilco) [reply custom-commands/hospitality/test/hospitality_skill.json:64]: Use the fake values I have here: https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/blob/master/clients/csharp-dotnet-core/voice-assistant-test/docs/json-templates/app-config-template.json

Comment (qfweiwei) [reply custom-commands/hospitality/test/hospitality_skill.json]: Update hospitality_skill.json with suggested fake values for SubscriptionKey and CustomCommandsAppId.

File: custom-commands/hospitality/test/input-files/Test_FallbackCommand_TextInput.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+[
+	{
+		"DialogID": "0",
+		"Description": "Testing 'turn it'",
+		"Skip": false,
+		"Turns": [
+			{
+				"TurnID": 0,
+				"Utterance": "turn it",
+				"WavFile": "",
```
Comment (dargilco) [inline custom-commands/hospitality/test/input-files/Test_FallbackCommand_TextInput.json:10]: Remove fields that are empty

File: custom-commands/hospitality/test/input-files/Test_FallbackCommand_TextInput.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+				"ExpectedResponses": [
+					{
+						"type": "message",
+						"text": "Sorry. I didn't quite catch that||I'm not sure what you meant by that",
```
Comment (dargilco) [inline custom-commands/hospitality/test/input-files/Test_FallbackCommand_TextInput.json:14]: Put a blank space before and after the || so it is easier to read

File: custom-commands/hospitality/test/input-files/Test_FallbackCommand_TextInput.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+						"speak": "<speak version=\"1.0\" xml:lang=\"EN-US\"><voice name=\"Microsoft Server Speech Text to Speech Voice (en-US, Jessa24kRUS)\">Sorry. I didn't quite catch that</voice></speak>||<speak version=\"1.0\" xml:lang=\"EN-US\"><voice name=\"Microsoft Server Speech Text to Speech Voice (en-US, Jessa24kRUS)\">I'm not sure what you meant by that</voice></speak>",
+						"inputHint": "acceptingInput"
+					}
+				],
+				"ExpectedTTSAudioResponseDurations": [
+					"3500||2500"
```
Comment (dargilco) [inline custom-commands/hospitality/test/input-files/Test_FallbackCommand_TextInput.json:20]: Can you also support space before and after the || here?

File: custom-commands/hospitality/test/input-files/Test_TurnOnOff_TextInput.json [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+						"type": "message",
+						"text": "Ok, turning the AC on",
+						"speak": "<speak version=\"1.0\" xml:lang=\"EN-US\"><voice name=\"Microsoft Server Speech Text to Speech Voice (en-US, Jessa24kRUS)\">Ok, turning the AC on</voice></speak>",
+						"inputHint": "acceptingInput"
+					}
+				],
+				"ExpectedTTSAudioResponseDuration": [
+					2800
+				],
+				"ExpectedResponseLatency": 2500
```
Comment (dargilco) [inline custom-commands/hospitality/test/input-files/Test_TurnOnOff_TextInput.json:39]: Any reason you are using different values of response latency? If not, can you use the same value in all tests? try to make it a small number.

File: custom-commands/hospitality/test/input-files/Test_FallbackCommand_TextInput.json (hunk 2 above)
Comment (dargilco) [reply custom-commands/hospitality/test/input-files/Test_FallbackCommand_TextInput.json:10]: I'm assuming your next PR will include tests with WAV file input?

File: clients/csharp-dotnet-core/voice-assistant-test/tool/DialogResult.cs [hunk 6]
```diff
@@ -177,10 +177,27 @@ public static int CompareJObjects(JObject expected, JObject actual, bool enableL
                         string actualResult = actualValue.ToString();
```
[the remark this answers is not in the document]
Comment (qfweiwei) [reply clients/csharp-dotnet-core/voice-assistant-test/tool/DialogResult.cs:177]: I use the same branch of last tool change PR for this hospitality test PR. That's why you can see already merged changes. I should use a new branch though.

There are 4 commits. The last 2 commits are for new added test files. Please review last 2 commits. Or I can cancel this PR, create a new branch, and commit again.

**Comment by qfweiwei** [comment]
Will submit a new PR for only hospitality skill tests.

**Pull request closed without merging**